config/webpack/webpack.common.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
             ],
         }),
 
+        new IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Comment:
No harm I suppose, but we'll need to remember to remove this when we replace Moment.

Reply:
Oh yeah, so I'm not sure we're removing `moment` entirely just removing it so it doesn't produce duplicates in the main JS bundle when importing our other libraries.

Reply:
The `locales` thing just seemed unused and decided to follow this guidance 

https://github.com/jmblog/how-to-optimize-momentjs-with-webpack

Reply:
Gonna remove some of this stuff for now and focus on the bundle improvements in another issue.